packages/config/config/devices/0x0258/nas-pd07z.json

```diff
@@ -10,63 +10,169 @@
 		}
 	],
 	"firmwareVersion": {
-		"min": "0.0",
+		"min": "2.6",
 		"max": "255.255"
 	},
 	"paramInformation": [
 		{
 			"#": "1",
-			"$import": "templates/shenzhen_neo_template.json#blink_on_motion"
```

Comment:
Also, please don't undo the template imports. They are done for consistency and to make some of the manufacturer-provided labels more easily understood.